src/cmd/ship.go

```diff
@@ -116,6 +116,7 @@ func getShipStepList(config shipConfig) (result steps.StepList) {
 	result.Append(&steps.CheckoutBranchStep{BranchName: branchToMergeInto})
 	canShipWithDriver, defaultCommitMessage := getCanShipWithDriver(config.BranchToShip, branchToMergeInto)
 	if canShipWithDriver {
+		result.Append(&steps.PushBranchStep{BranchName: config.BranchToShip})
```

Comment:
If there are any conflicts, we need to update the remote branch before we merge via the API as otherwise the remote doesn't have the resolutions.